_drafts/wework-com-is-going-static.markdown

```diff
@@ -0,0 +1,114 @@
+
+```coffeescript
+extensions: [
+  records(
+    marketsByCountry: {
+      url:
+        path: "#{process.env.API_ENDPOINT}/api/to/locations/data"
+    }
+  )
+]
+```
+
+And here is how this data is referenced inside of our `jade` templates:
+
+```jade
+each countries, index in records.marketsByCountry
+  .row
+    .col-12.col
+      h4
+        != countries[0]
+```
+
+Once we are ready to move our individual location pages over to *static*, all we need to do is pass a `template` option to the `records` block in our app.coffee config, and Roots will generate the individual location pages when it builds the site. This would look something like this:
+
+```coffeescript
+extensions: [
+  records(
+    marketsByCountry: {
+      url:
+        path: "#{process.env.API_ENDPOINT}/api/to/locations/data"
+      template: "views/_location.jade"
+      out: (location) -> "/locations/#{location.market.slug}/#{location.slug}"
+    }
+  )
+]
+```
+
+
+### Netlify: The static host with the most
+
+One of the biggest challenges of this whole process has been to figure out a way to slowly migrate our site over from a dynamic app, over to a static one. One way to do this would be to stop all development for a few months and rebuild the entire site using this new infrastructure. But as the saying goes, "ain't nobody got time for that."
+
+We needed a way to move things over piecemeal. One page at a time. This meant that we couldn't move wework.com to a new static host all at once, but at the same time, we needed some URL's to serve up the old content, and some URL's to serve up the new static pages. One way to do this is via a [reverse proxy](https://en.wikipedia.org/wiki/Reverse_proxy).
+
+There are many different ways to skin this cat, ranging from installing and configuring your own Apache or Ngnix server, using something like rack-reverse-proxy to a slew of other similar solutions, each with their own pros and cons. For us, being a team of web developers, we wanted to spend our time focusing on our KPI's and optimizing our pages for performance, and not on setting up and managing servers. We had a few requirements. Specifically, we were looking for a host that could:
```

Comment:
I think this might be too transparent. I would change to something like "ranging from Apache or Nginx configuration, using something like... ...our pages for performance, and not on configuring servers"